api/types/swarm/swarm.go

```diff
@@ -10,6 +13,8 @@ type ClusterInfo struct {
 	Spec                   Spec
 	TLSInfo                TLSInfo
 	RootRotationInProgress bool
+	DefaultAddrPool        []*net.IPNet
+	SubnetSize             int
```

Comment:
See my other comment; why `int` instead of `uint32` (if that's what we want?)

Reply:
modified it